astroquery/alma/core.py

```diff
@@ -234,9 +234,16 @@ def stage_data(self, uids):
         self._staging_log['initial_response'] = response
         log.debug("First response URL: {0}".format(response.url))
         if response.status_code == 405:
-            raise HTTPError("Received an error 405: this may indicate you "
-                            "have already staged the data.  Try downloading "
-                            "the file URLs directly with download_files.")
+            if hasattr(self,'_last_successful_staging_log'):
+                print("Error 405 received.  If you have previously staged "
```

Comment:
I think a `log.info` message would be more appropriate then `print` but I don't know how you handled this in the package.